deepchem/models/tensorgraph/tests/test_layers.py

```diff
@@ -1,61 +1,51 @@
-from deepchem.models.tensorgraph.layers import CombineMeanStd
-from deepchem.models.tensorgraph.layers import Repeat
-from deepchem.models.tensorgraph.layers import Gather
 from deepchem.models.tensorgraph.layers import GRU
-from deepchem.models.tensorgraph.layers import TimeSeriesDense
+from deepchem.models.tensorgraph.layers import Gather
+from deepchem.models.tensorgraph.layers import GraphConv
+from deepchem.models.tensorgraph.layers import GraphGather
 from deepchem.models.tensorgraph.layers import Input
+from deepchem.models.tensorgraph.layers import InputFifoQueue
+from deepchem.models.tensorgraph.layers import InteratomicL2Distances
+from deepchem.models.tensorgraph.layers import IterRefLSTMEmbedding
 from deepchem.models.tensorgraph.layers import L2Loss
-from deepchem.models.tensorgraph.layers import Concat
-from deepchem.models.tensorgraph.layers import Constant
-from deepchem.models.tensorgraph.layers import Variable
-from deepchem.models.tensorgraph.layers import Add
-from deepchem.models.tensorgraph.layers import Multiply
+from deepchem.models.tensorgraph.layers import LSTMStep
+from deepchem.models.tensorgraph.layers import LayerSplitter
 from deepchem.models.tensorgraph.layers import Log
-from deepchem.models.tensorgraph.layers import InteratomicL2Distances
-from deepchem.models.tensorgraph.layers import SoftMaxCrossEntropy
+from deepchem.models.tensorgraph.layers import Multiply
 from deepchem.models.tensorgraph.layers import ReduceMean
-from deepchem.models.tensorgraph.layers import ToFloat
-from deepchem.models.tensorgraph.layers import ReduceSum
 from deepchem.models.tensorgraph.layers import ReduceSquareDifference
-from deepchem.models.tensorgraph.layers import Conv2D
-from deepchem.models.tensorgraph.layers import Conv3D
-from deepchem.models.tensorgraph.layers import MaxPool
-from deepchem.models.tensorgraph.layers import MaxPool3D
-from deepchem.models.tensorgraph.layers import InputFifoQueue
-from deepchem.models.tensorgraph.layers import GraphConv
-from deepchem.models.tensorgraph.layers import GraphPool
-from deepchem.models.tensorgraph.layers import GraphGather
-from deepchem.models.tensorgraph.layers import BatchNorm
+from deepchem.models.tensorgraph.layers import ReduceSum
+from deepchem.models.tensorgraph.layers import Repeat
+from deepchem.models.tensorgraph.layers import Reshape
+from deepchem.models.tensorgraph.layers import SluiceLoss
 from deepchem.models.tensorgraph.layers import SoftMax
-from deepchem.models.tensorgraph.layers import WeightedError
+from deepchem.models.tensorgraph.layers import SoftMaxCrossEntropy
+from deepchem.models.tensorgraph.layers import TensorWrapper
+from deepchem.models.tensorgraph.layers import TimeSeriesDense
+from deepchem.models.tensorgraph.layers import ToFloat
+from deepchem.models.tensorgraph.layers import Transpose
+from deepchem.models.tensorgraph.layers import Variable
 from deepchem.models.tensorgraph.layers import VinaFreeEnergy
```

Comment:
I'm a fan of alphabetic imports! We should see if we can make yapf figure out how to do this for us...